src/utils/ipv6.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Compares if IPV6 addresses are the same.
+ * @param FirstIp First IP to compare with
+ * @param SecondIp Second IP to compare with
+ */
+export function compareIPV6(FirstIp: string, SecondIp: string) {
```

Comment:
There is a convention, that only class names start from Upper Case. The exception is for JSX components.